src/jobs/build-system.yml

```diff
@@ -22,14 +19,14 @@ steps:
       version: $ELIXIR_VERSION
   - install-hex-rebar
   - install-nerves-bootstrap
-  - br2-primary-site
```

Comment:
[br2-primary-site.yml](https://github.com/nerves-project/build-tools/blob/6c3fd0e3322d1cf3e260016f538e14ea88ed8927/src/commands/br2-primary-site.yml)

I'm not sure if there are any side effects to removing this. It wasn't obvious to me why the approach was taken to use CI artifacts rather than the cache, and I may have overlooked something. It seems to work fine for the `rpi3a` system though.

Reply:
I think you're right. Using a cache effectively replaces this. Frank might be able to say more

Reply:
I have mixed opinions on this. The first one is that I want this fixed, and this seems like a practical way to do it. The second one is that using CircleCI's cache doesn't address the problem of archiving Buildroot and Nerves packages. So I'm really grateful to have this fix even if I don't think it's a long term solution.

Reply:
Oh, hang on, I'm late to reviewing. You've updated this to push to S3. I need to read on...

Reply:
Follow-up: This will be handled by setting `BR2_PRIMARY_SITE` to the Nerves download site, rather than to a CircleCI build artifact.